### PR TITLE
update develop for release 0.14.0

### PR DESCRIPTION
**Pull request by cqc-melf**

### CODE REVIEW

File: modules/pytket-stim/docs/changelog.rst [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Changelog
 ~~~~~~~~~
 
+0.2.0 (September 2021)
+----------------------
+
+* Updated qulacs version requirement to 0.3.
```
Comment (cqc-alec) [inline modules/pytket-stim/docs/changelog.rst:7]:
```suggestion
* Updated pytket version requirement to 0.14.
```

File: modules/pytket-qulacs/docs/changelog.rst [hunk 2]
```diff
@@ -1,8 +1,8 @@
 Changelog
 ~~~~~~~~~
 
-0.11.0 (unreleased)
--------------------
+0.11.0 (September 2021)
+-----------------------
```
Comment (cqc-alec) [inline modules/pytket-qulacs/docs/changelog.rst:5]: Need to update changelog and `setup.py` for pytket-qulacs.

**Review by cqc-alec — CHANGES_REQUESTED** [review]

**Comment by cqc-melf** [comment]
Thank you for spotting that. I have changed that.

Have you seen the qsharp windows problem?

https://github.com/CQCL/pytket-extensions/runs/3530754744?check_suite_focus=true

**Comment by cqc-alec** [comment]
> Thank you for spotting that. I have changed that.
> 
> Have you seen the qsharp windows problem?
> 
> https://github.com/CQCL/pytket-extensions/runs/3530754744?check_suite_focus=true

Don't know what the issue is here; it looks a bit like https://github.com/microsoft/iqsharp/issues/488 which was fixed by upgrading my qsharp-core version locally.

File: modules/pytket-qsharp/setup.py [hunk 3]
```diff
@@ -39,9 +39,9 @@
     packages=find_namespace_packages(include=["pytket.*"]),
     include_package_data=True,
     install_requires=[
-        "pytket ~= 0.14.0rc0",
-        "qsharp ~= 0.18.2107",
-        "qsharp-core ~= 0.18.2107",
+        "pytket ~= 0.14.0",
+        "qsharp == 0.18.2107",
+        "qsharp-core == 0.18.2107",
```
Comment (cqc-alec) [inline modules/pytket-qsharp/setup.py:44]: Try `~=0.18.2107,<0.18.2108` ?

**Comment by cqc-melf** [comment]
Thank you for your help! I am not sure if that is really the same issue, but I wanted to try make sure there is no problem in the version.

**Comment by cqc-alec** [comment]
Might be worth looking for the last good run of the qsharp tests on the CI and scanning the logs for versions of relevant modules. Or asking someone with a Windows machine to test it.

File: .github/workflows/build-test [hunk 4]
```diff
@@ -48,21 +48,25 @@ then
     python -m pip install --upgrade mypy
 fi
 for MODULE in $MODULES_TO_TEST; do
-    echo "${MODULE}..."
 
-    cd ${MODULEDIR}/${MODULE}/tests
+    if [[ "${MODULE}" != "pytket-qsharp" || "${PLAT}" != "Windows" ]]
```
Comment (cqc-alec) [inline .github/workflows/build-test:52]: Can you please raise an issue on Github, and insert a FIXME comment here referencing the issue?

**Review by cqc-alec — CHANGES_REQUESTED** [review]

File: .github/workflows/build-test (hunk 4 above)
Comment (cqc-melf) [reply .github/workflows/build-test:52]: Yes

Comment (cqc-melf) [reply .github/workflows/build-test:53]: https://github.com/microsoft/iqsharp/issues/512

Comment (cqc-alec) [reply .github/workflows/build-test:53]: I actually meant on _this_ repo. Could you create an issue on pytket-extensions and reference it here as well?

**Review by cqc-alec — CHANGES_REQUESTED** [review]

File: .github/workflows/build-test (hunk 4 above)
Comment (cqc-melf) [reply .github/workflows/build-test:55]: Sorry, I misunderstood that. I have addd a new one for pytket-extensions:
https://github.com/CQCL/pytket-extensions/issues/180

**Review by cqc-alec — APPROVED** [review]

**Pull request merged**